src/webgpu/web_platform/copyToTexture/canvas.spec.ts

```diff
@@ -2,13 +2,234 @@ export const description = `
 copyToTexture with HTMLCanvasElement and OffscreenCanvas sources.
 
 - x= {HTMLCanvasElement, OffscreenCanvas}
-- x= {2d, webgl, webgl2, gpupresent, bitmaprenderer} context, {various context creation attributes}
+- x= {2d, webgl, webgl2} context, {various context creation attributes}
```

Comment:
I think these two lines can be removed since I think they're covered in the description of copy_contents_from_canvas now.